jerry-core/lit/lit-char-helpers.h

```diff
@@ -220,4 +220,13 @@ extern uint32_t lit_char_hex_to_int (ecma_char_t);
  */
 extern bool lit_char_is_word_char (ecma_char_t);
 
+/*
+ * Utility functions for uppercasing / lowercasing
+ */
+
+#define MAXIMUM_OTHERCASE_LENGTH (3)
```

Comment:
@zherczeg, could you, please, add `LIT_` prefix and add describing comment for the definition?